README.md

```diff
@@ -1,14 +1,11 @@
 # Marlin 3D Printer Firmware
 
-[![Build Status](https://travis-ci.org/MarlinFirmware/Marlin.svg?branch=RCBugFix)](https://travis-ci.org/MarlinFirmware/Marlin)
-[![Coverity Scan Build Status](https://scan.coverity.com/projects/2224/badge.svg)](https://scan.coverity.com/projects/2224)
-
```

Comment:
Fix the things I pointed out, commit the changes to your branch, and push them to "origin" to update the PR. If you're not sure how to manage a PR please read the Github documentation.